ipaserver/install/dogtaginstance.py

```diff
@@ -308,11 +308,12 @@ def secure_ajp_connector(self):
         doc = server_xml.getroot()
 
         # no AJP connector means no need to update anything
-        connectors = doc.xpath('//Connector[@port="8009"]')
+        connectors = doc.xpath('//Connector[@protocol="AJP/1.3"]')
```

Comment:
Please explain why you are changing the search criteria. I assume this is because it is more specific to this version of AJP and relying only on the port could lead to future problems but this is only a wild guess.

Reply:
AJP is a protocol a connector can have. Specifically it has `AJP/1.3` in the `protocol=` field. Most commonly it listens on port 8009. However, you can run AJP on any port you desire. The comment above this block of code says "AJP connector", so we should update the AJP connector, not whatever is listening on port 8009.

Reply:
Yes, exactly! It should be in the commit message.

Reply:
ACK, adding.